cvat-core/tests/internal/filter.js.bkp

```diff
@@ -0,0 +1,121 @@
+// Copyright (C) 2020-2021 Intel Corporation
```

Comment:
Need to update these test, not just drop them out. We can do it in separated PR, but in this case, please, create an issue.

Reply:
Will create an issue

Reply:
Okay. This ``.bpk`` file probably should be removed since we have all these files on git, right?

Reply:
OK